vllm/worker/cpu_worker.py

```diff
@@ -177,6 +177,9 @@ def init_device(self) -> None:
 
     def load_model(self):
         self.model_runner.load_model()
+        logger.info("Warming up...")
```

Comment:
is this for debug?

Reply:
just printing some status messages, because the compilation may take a while.

Reply:
Can we add `The compilation may take several minutes.` or something like that?